README.md

```diff
@@ -210,6 +214,16 @@ annotation to your .proto file
    ```
 
 ## Parameters and flags
+
+During code generation with `protoc`, flags to grpc-gateway tools must be passed
+through protoc using the `--<tool_suffix>_out=<flags>:<path>` pattern, for
```

Comment:
This is something I struggled with a fair bit when getting started, and I expect other new users might too.

Reply:
Great addition!